src/overlays/kaleido_scope/ovl_kaleido_scope/z_kaleido_scope_NES.c

```diff
@@ -633,19 +747,413 @@ TexturePtr D_8082B9A8[] = {
     gPauseToMasksENGTex,
     gPauseToSelectItemENGTex,
 };
+void KaleidoScope_DrawInfoPanel(PlayState* play) {
+    static s16 sPauseLRCursorColorTargets[][4] = {
+        { 180, 210, 255, 220 },
+        { 100, 100, 150, 220 },
+    };
+    static s16 sPauseLRCursorColorTimer = 20;
+    static s16 sPauseLRCursorColorIndex = 0;
+    static s16 sPauseLRCursorRed;
+    static s16 sPauseLRCursorGreen;
+    static s16 sPauseLRCursorBlue;
+    static s16 sPauseLRCursorAlpha;
```

Comment:
And, if I'm understanding the info panel comment from above correctly, should these be "ZR" instead of "LR"?